demos/frontend-react/Review.md

```diff
@@ -0,0 +1,39 @@
+# React.js review
```

Comment:
This file should be called as in templates/, i.e. either Review.md or review.md but not both